src/Native/Unix/configure.cmake

```diff
@@ -467,11 +467,19 @@ check_cxx_source_compiles(
     HAVE_TCP_VAR_H
 )
 
+check_include_files(
+    sys/cdefs.h
+    HAVE_SYS_CDEFS_H)
+
+if (HAVE_SYS_CDEFS_H)
+    set(EXTRA_INCLUDE_FILES "#include <sys/cdefs.h>")
+endif()
+
 # If sys/cdefs is not included on Android, this check will fail because
 # __BEGIN_DECLS is not defined
 check_cxx_source_compiles(
     "
-    #include <sys/cdefs.h>
+    ${EXTRA_INCLUDE_FILES}
```

Comment:
Ok. If that can't be made to work, let's at least choose a more specific name instead of EXTRA_INCLUDE_FILES, like SYS_CDEFS_INCLUDE_FILE, or something along those lines.